source/persistency/PetaloPersistencyManager.cc

```diff
@@ -34,7 +35,7 @@
 #include <iostream>
 #include <iomanip>
 
-#include "CLHEP/Units/SystemOfUnits.h"
+#include <CLHEP/Units/SystemOfUnits.h>
```

Comment:
What's the difference between these two lines?

Reply:
Nothing, it's just a matter of convention.